### ISSUE TITLE
Devicons crashed 

### ISSUE THREAD

**Issue by bsjaramillo** [equalsraf/neovim-qt#873]
Icons are in chinese, lol.
![image](https://user-images.githubusercontent.com/33032851/124400661-cb3a0500-dce9-11eb-952d-078aa27fdcf6.png)

Why ? i had not been use neovim-qt for a long time. Nvim is right.

**Comment by jgehrig** [comment]
Sorry, this isn't enough information to help you...

What version of `neovim-qt`? `nvim-qt --version`

What fonts are you using?

Is there a minimal configuration that reproduces this issue? Ex) `nvim-qt -- -U minimal.vim`

**Comment by bsjaramillo** [comment]
The version is:
![image](https://user-images.githubusercontent.com/33032851/124494371-5ebc1600-dd7c-11eb-913e-ac5a1ae78095.png)

Yes, minimal configuration reproduces that error too.
I use DankMono or Cascadia Code fonts patched. When i open nvim-qt it show a message error too fast about the font, but i deactivate the fonts patched and i have the same issue.

**Comment by bsjaramillo** [comment]
I installed the lastest version, and i have the same issue also disable the fonts; now i dont have any error message about the font. 
![image](https://user-images.githubusercontent.com/33032851/124507853-4c4dd680-dd94-11eb-9151-145e8ef48d48.png)

**Comment by jgehrig** [comment]
It is hard to follow what you're doing. Please provide a clear configuration and set of steps.

Can you provide a `minimal.vim` file for me to load with `nvim-qt -- -u minimal`?

> I use DankMono or Cascadia Code fonts patched. When i open nvim-qt it show a message error too fast about the font, but i deactivate the fonts patched and i have the same issue.

Try manually loading the font: `:GuiFont DankMono:h11`.

I'd guess the error is one of these:
`{Font Name} is not a fixed pitch Font`
` Warning: Font {Font Name} reports bad fixed pitch metrics`

Do you have `:GuiRenderLigatures 1`?

Does this issue reproduce with a stock Windows font such as Consolas?

**Comment by bsjaramillo** [comment]
Hi, i found the bug, the problem was Cascadia Code font, i did change for another font. I had been try this before all already, but it wasn't working, also i use this one font in console for nvim and it was always works fine, i dont know what happend, sorry.
